### ISSUE TITLE
unable to share /srv/backups folder in syncthing container 

### ISSUE THREAD

**Issue by alpeshgajbe** [gnowledge/gstudio-docker#6]
When I tried to share /srv/backups folder it gave me a permission denied error.
The simple workaround was to just run 
touch /srv/backups/.stfolder

**Comment by gnowgi** [comment]
yes.  If the syncthing user can write in the folder then it creates it automatically.  since the folder is mounted through docker, it is owned by root, so syncthing cannot create the marker file.
